media/css/_faculty_structure.scss

```diff
@@ -0,0 +1,212 @@
+.bg-structure {
+  position: relative;
+  width: 890px;
+  margin: 0 auto;
+  height: 1010px;
+  padding: 5px;
+  background-image: url('../img/line.png');
+  background-position: top -15px center;
+  background-repeat: no-repeat;
+}
+
+.struct-elem {
+  display: inline-block;
```

Comment:
Properties should be ordered background, border, display, font-size, padding, position, text-align